### PR TITLE
fix: Adding Slack and k-dev channel link

### PR DESCRIPTION
**Pull request by vishakhanihore**
- Channel and Slack link was missing.

### CODE REVIEW

**Comment by vishakhanihore** [comment]
cc: @parispittman
/assign @mrbobbytables

File: content/_index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ It is intended to be the hub for all things related to the Kubernetes Contributo
 
 The first step is to check out the [contributor guide](/guide/). All contributors must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
 
-If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack] and pop over to the [#kubernetes-dev channel][kdev].
+If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack]((https://slack.k8s.io/)) and pop over to the [#kubernetes-dev channel](https://app.slack.com/client/T09NY5SBT/C09R23FHP).
```
Comment (mrbobbytables) [inline content/_index.md]: Is there a reason you switched from the reference style links? In most cases they're preferred over inline.

**Comment by vishakhanihore** [comment]
They are not rendering

**Comment by mrbobbytables** [comment]
They're rendering correctly in BlackFriday markdown (which was why it wasn't noticed), the issue with them in the GitHub markdown is there isn't a line separating the last list item and the link.

From https://kubernetes-contributor.netlify.com/
Screen Shot 2020-01-09 at 6 42 36 AM


```
* Workshop videos
[Slack]: https://slack.k8s.io/
[kdev]: https://app.slack.com/client/T09NY5SBT/C09R23FHP
```

Should be

```
* Workshop videos

[Slack]: https://slack.k8s.io/
[kdev]: https://app.slack.com/client/T09NY5SBT/C09R23FHP
```

It looks like the rest of the doc uses inline links, so this is fine to align usage. In general though, reference links are much preferred. It makes it a little more work when writing markdown but it is much more readable for others to review and in the event of a link change in the future it's easy to change it without disrupting the content itself.

File: content/_index.md (hunk 1 above)
Comment (mrbobbytables) [inline content/_index.md]: The double `
```suggestion
If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack](https://slack.k8s.io/) and pop over to the [#kubernetes-dev channel](https://app.slack.com/client/T09NY5SBT/C09R23FHP).
```
The dobule `(( ))` doesn't render the link correctly in GH markdown. :x

**Comment by vishakhanihore** [comment]
> They're rendering correctly in BlackFriday markdown (which was why it wasn't noticed), the issue with them in the GitHub markdown is there isn't a line separating the last list item and the link.
> 
> From https://kubernetes-contributor.netlify.com/
> Screen Shot 2020-01-09 at 6 42 36 AM
> 
> ```
> * Workshop videos
> [Slack]: https://slack.k8s.io/
> [kdev]: https://app.slack.com/client/T09NY5SBT/C09R23FHP
> ```
> 
> Should be
> 
> ```
> * Workshop videos
> 
> [Slack]: https://slack.k8s.io/
> [kdev]: https://app.slack.com/client/T09NY5SBT/C09R23FHP
> ```
> 
> It looks like the rest of the doc uses inline links, so this is fine to align usage. In general though, reference links are much preferred. It makes it a little more work when writing markdown but it is much more readable for others to review and in the event of a link change in the future it's easy to change it without disrupting the content itself.

So the formatting was all the issue. I will be making ASAP !

**Comment by vishakhanihore** [comment]
@mrbobbytables Done the changes, Please have a look !

File: content/_index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Remember: No question is too small.
 There are many ways you can stay involved with the Kubernetes Contributor community. 
 
 * Join a [SIG](https://git.k8s.io/community/sig-list.md)
-* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide
+* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide(https://git.k8s.io/community/contributors/guide)
```
Comment (mrbobbytables) [inline content/_index.md]:
```suggestion
* Contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and [contributor guide](/guide/)
```
If you're adding the link to the guide, I would have it refer to the version of the guide that is hosted on the site like the link above  in the [Get Involved](https://github.com/kubernetes-sigs/contributor-site/blob/master/content/_index.md#get-involved) section. 👍

**Comment by vishakhanihore** [comment]
@mrbobbytables Made the changes. Please review and merge

**Comment by mrbobbytables** [comment]
/lgtm
/approve
🤦‍♂ still kicking myself for the other one

**Comment by mrbobbytables** [comment]
woops, accidentally removed the label
/approve

**Pull request merged**